charm/tests/unit/test_charm.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Unit tests for NetBox charm."""
+
+
+async def test_empty() -> None:
```

Comment:
Is this necessary?

Reply:
Unfortunately, without unit test, coverage fails. As I plan to add them in the next PR, I preferred to have that dummy test for now to have the CI/CD well configured.